HHS/OPRE-OPS
# feat: ✨ CANs List

## What changed

- adds the CANs list and friends
  - CANTable and CANTableRow
- adds pagination
- adds tooltips
- adds type definition files 

## Issue

#346 #2137 

## How to test

1. navigate to `/cans`
2. see cans list
3. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/cc418ef9-e84d-4d27-90f5-e2c5cebbebce

## Definition of Done Checklist
